acceptance/tests/base/dsl/helpers/host_helpers_test.rb

```diff
@@ -0,0 +1,1377 @@
+
+      scp_to hosts.first, local_filename, remote_tmpdir
+
+      remote_filename = File.join(remote_tmpdir, "testfile.txt")
+      remote_contents = on(hosts.first, "cat #{remote_filename}").stdout
+      assert_equal "contents\n", remote_contents
+    end
+  end
+
+  step "#scp_to creates the file on all remote systems when a host array is provided" do
+    Dir.mktmpdir do |local_dir|
+      local_filename = File.join(local_dir, "testfile.txt")
+      File.open(local_filename, "w") do |local_file|
+        local_file.puts "contents"
+      end
+
+      remote_tmpdir = tmpdir_on hosts.first
+      on hosts, "mkdir -p #{remote_tmpdir}"
+      remote_filename = File.join(remote_tmpdir, "testfile.txt")
+
+      scp_to hosts, local_filename, remote_tmpdir
+
+      hosts.each do |host|
+        remote_contents = on(host, "cat #{remote_filename}").stdout
+        assert_equal "contents\n", remote_contents
+      end
+    end
+  end
+
+  if test_scp_error_on_close?
+    step "#scp_from fails if the local path cannot be found" do
+      remote_tmpdir = tmpdir_on hosts.first
+      remote_filename = File.join(remote_tmpdir, "testfile.txt")
+      on hosts.first, %Q{echo "contents" > #{remote_filename}}
+      assert_raises Beaker::Host::CommandFailure do
+        scp_from hosts.first, remote_filename, "/non/existent/file.txt"
+      end
+    end
+
+    step "#scp_from fails if the remote file cannot be found" do
+      Dir.mktmpdir do |local_dir|
+        assert_raises Beaker::Host::CommandFailure do
+          scp_from hosts.first, "/non/existent/remote/file.txt", local_dir
+        end
+      end
+    end
+  end
+
+  step "#scp_from creates the file on the local system" do
+    Dir.mktmpdir do |local_dir|
+      remote_tmpdir = tmpdir_on hosts.first
+      remote_filename = File.join(remote_tmpdir, "testfile.txt")
+      on hosts.first, %Q{echo "contents" > #{remote_filename}}
+
+      scp_from hosts.first, remote_filename, local_dir
+
+      local_filename = File.join(local_dir, "testfile.txt")
+      assert_equal "contents\n", File.read(local_filename)
+    end
+  end
+
+  step "#scp_from CURRENTLY creates and repeatedly overwrites the file on the local system when given a hosts array" do
+    # NOTE: expect this behavior to be well-documented, or for overwriting a
+    #       file repeatedly to generate an error
+
+    Dir.mktmpdir do |local_dir|
+      local_filename = File.join(local_dir, "testfile.txt")
+      remote_tmpdir = tmpdir_on hosts.first
+      remote_filename = File.join(remote_tmpdir, "testfile.txt")
+      on hosts, "mkdir -p #{remote_tmpdir}"
+      results = on hosts, %Q{echo "${RANDOM}:${RANDOM}:${RANDOM}" > #{remote_filename}}
+
+      scp_from hosts, remote_filename, local_dir
+
+      remote_contents = on(hosts.last, "cat #{remote_filename}").stdout
+      local_contents = File.read(local_filename)
+      assert_equal remote_contents, local_contents
+    end
+  end
+
+  if hosts.first.is_cygwin?
```

Comment:
is_cygwin? again.  I'll stop commenting on it, but this workflow should be fixed.